modules/minimint-ln/src/lib.rs

```diff
@@ -526,57 +526,56 @@ impl FederationModule for LightningModule {
             ApiEndpoint {
                 path_spec: "/account/:contract_id",
                 params: &["contract_id"],
-                method: http::Method::Get,
+                method: http::Method::GET,
                 handler: |module, params, _body| {
                     let contract_id: ContractId = match params
                         .get("contract_id")
                         .expect("Contract id not supplied")
                         .parse()
                     {
                         Ok(id) => id,
-                        Err(_) => return Ok(http::Response::new(400)),
+                        Err(_) => return Ok(http::StatusCode::BAD_REQUEST.into()),
                     };
 
-                    let contract_account = module
-                        .get_contract_account(contract_id)
-                        .ok_or_else(|| http::Error::from_str(404, "Not found"))?;
+                    let contract_account = match module.get_contract_account(contract_id) {
+                        Some(c) => c,
+                        None => return Ok(http::StatusCode::NOT_FOUND.into()),
```

Comment:
Same here: now returnes `Ok(...)` instead of `Err(...)`  previously we'd get [this](https://github.com/fedimint/minimint/pull/197/files#diff-a374ffabf92feaab4cee76c1d04e4555dcfa8e22d1b7a59f908c8079c488fa9dR112-R114)